src/components/formGeneratorCustom/PlacesAutoCompleteBox/PlacesAutoComplete.vue

```diff
@@ -110,20 +96,41 @@
       },
     },
     computed: {
-      ...mapState('places-auto-complete', {placeloading: 'isFindPending'}),
-      ...mapState('geocode', {geoloading: 'isFindPending'}),
-      ...mapGetters('places-auto-complete', {addresses: 'list'}),
+      ...mapState(usePlacesAutoComplete, {placeloading: 'isFindPending'}),
+      ...mapState(useGeocode, {geoloading: 'isFindPending'}),
+      ...mapState(usePlacesAutoComplete, {addresses: 'items'}),
```

Comment:
you did mapState(usePlacesAutoComplete twice combine them.

Reply:
oh... resolved